src/assets/stylesheets/styles.scss

```diff
@@ -108,6 +108,67 @@
   word-break: break-all;
 }
 
+// NavGlobal
+#nav-global {
+  background-color: color('blue-60');
```

Comment:
Non-blocking suggestion: I think we could remove the inner `div` if we consolidated these two elements’ background properties into one, like:

`background: color('blue-60') linear-gradient(0deg, rgba(0, 0, 0, 0) 25%, rgba(0, 0, 0, 0.2) 100%);`